projects/go-lib/src/styles/_forms.scss

```diff
@@ -106,6 +113,66 @@ $input--disabled-background: rgba($base-dark-secondary, .4);
   }
 }
 
+/* Hide the default checkbox */
+.go-form__checkbox--hidden {
+  cursor: pointer;
+  height: 0;
+  opacity: 0;
+  position: absolute;
+  width: 0;
+}
+
+/* Create a custom checkbox */
+.go-form__custom-checkbox {
+  background-color: $base-light;
+  border: 1px solid lighten($base-light-secondary, 13.5%);
+  border-radius: 2px;
+  display: inline-block;
+  height: 0.875rem;
+  left: 0;
+  position: absolute;
+  top: 0.1rem;
+  width: 0.875rem;
+}
+
+/* When the checkbox is checked, change the background and border color of the custom checkbox */
+.go-form__checkbox:checked ~ .go-form__custom-checkbox {
+  background-color: $ui-color-positive;
+  border-color: $ui-color-positive;
+}
+
+/* When the checkbox is disabled, change the background color of the custom checkbox */
+.go-form__checkbox:disabled ~ .go-form__custom-checkbox {
+  background-color: $theme-light-app-bg;
+}
+
+/* When the checkbox is disabled and checked, change the background and border color of the custom checkbox */
+.go-form__checkbox:disabled:checked ~ .go-form__custom-checkbox {
+  background-color: lighten($base-light-secondary, 13.5%);
+  border: 1px solid lighten($base-light-secondary, 13.5%);
+}
+
+/* Create and style a custom checkmark (hidden when not checked) */
+.go-form__custom-checkbox:after {
+  border: solid #FDFDFD;
```

Comment:
I believe we need to create a variable for this, unless it is equivalent to one of our calculated colors.

Reply:
I think this should just be `$base-light`. If you look at the DSM it says `or Shuttle` and `Shuttle` is just `#fff` which is `$base-light`.